### ISSUE TITLE
Add feature to remove inline image attachments

### ISSUE THREAD

**Issue by mandresen** [dsoares/Roundcube-Plugin-RemoveAttachments#7]
Would it be possible to expand the plugin's feature set to include removing inline image attachments?

**Comment by dsoares** [comment]
Like having one button to remove all inline images in the message body?

**Comment by mandresen** [comment]
Yes, "remove all" or buttons for the individual inlined images. 

The inline images don't show up as regular attachments that can be deleted, so it would be great to have a way of deleting these sometimes rather large inlined images.

Thanks for a nice plugin.

**Comment by mckaygerhard** [comment]
just for curious: why remove a message from a mail that already are in the inbox with those attachments already inside?

**Comment by mandresen** [comment]
I realize this probably comes down to personal preferences for keeping a tidy mail archive.

For some mail threads I'll receive lots of very large images, that will sometime appear as inline images instead of regular attachments. Personally, I like keeping a mail archive of whom I corresponded with – and when – but I don't need hundreds of MBs of images in a mail thread. I'll save them out for either further work in an image editing program, or perhaps save the photos in a photo archive for other reasons. This way one would be able to preserve the mail threads, and accompanying discussions, but no need to have a huge IMAP store to sync to all devices with old "photos from event X".

**Comment by mckaygerhard** [comment]
that its a interesting idea .. 

i think as i suppose, mayve @dsoares , @mandresen or whatevere made it, 
 only applies for smart people... (i mean, take apart the "personal purpose") 

i ask due in my case, i used for better sync so the history mails loads more faster

**Comment by numericOverflow** [comment]
+1,000 on this.  I installed the plugin hoping it did exactly as @mandresen suggested.  I like the ability to remove attachments from existing mail, but also often need to remove all attachments from forwarding/replying messages.  

Many business contacts cram all sorts of junk images on every message, and I don't want to send the attachments back when replying, so I have to delete them one-by-one.

A "remove all attachments" button on the send message screen would be awesome
![2018-12-19_170831](https://user-images.githubusercontent.com/3429424/50253796-21a3fa00-03b1-11e9-819a-fa0116f21662.jpg)

**Comment by numericOverflow** [comment]
Any progress updates?

**Comment by dsoares** [comment]
Sorry for the late answer.

@numericOverflow as for the "Remove all attachments", it already exists (it's the second button in the screenshot):
![Screen Shot 2020-11-29 at 22 57 23](https://user-images.githubusercontent.com/673736/100555911-7a1aec00-3296-11eb-8253-792ee26ed27d.png)

OK, it does not exist... only now i understand what you want: to be able to remove attachments from a message in the compose screen. But this plugin was not meant to do that, its purpose is to remove attachments from messages in your mailbox to save space. Never thought about a button like what you want, maybe i'll implement it.

@mandresen what you want will take a bit of time. I'll let this issue open, i'll see if i can do it this week.
